config/locales/es.yml

```
@@ -1516,18 +1515,13 @@ es:
       df_transfer_issue_message:
         email:
           body: |
-            Hello, You may have experienced an issue transferring your federal return data from IRS Direct File. This issue is now resolved.
+            Hola. Es posible que hayas tenido problemas transfiriendo los datos de tu declaración federal desde IRS Direct File. Se resolvió el problema.
 
-            To continue filing your state tax return, please create a new account with FileYourStateTaxes by visiting: https://www.fileyourstatetaxes.org/en/%{state_code}/landing-page
+            Para continuar con la preparación de tu declaración estatal, por favor crea una cuenta nueva con FileYourStateTaxes, visitando: https://www.fileyourstatetaxes.org/en/%{state_code}/landing-page
```

There are alot of links in transifex that are set to the english version I think because the link tool is hard to use
```suggestion
            Para continuar con la preparación de tu declaración estatal, por favor crea una cuenta nueva con FileYourStateTaxes, visitando: https://www.fileyourstatetaxes.org/es/%{state_code}/landing-page
```